layouts/partials/footer.html

```diff
@@ -0,0 +1,3 @@
+<footer>
+  <p>&copy 1999-{{ now.Format "2006" }}, Rice University. Except where otherwise noted, content on this site was created in collaboration with Illustrative Mathematics and is licensed under a <a href="https://creativecommons.org/licenses/by/4.0/" target="_blank" rel="noreferrer">Creative Commons Attribution 4.0 International License</a>. </p>
```

Comment:
Just as a note: This was :heavy_check_mark: by @mjoss17 and is slightly different than the early demo.